package.json

```diff
@@ -12,7 +12,11 @@
     "./fluent-gecko"
   ],
   "scripts": {
+    "predist": "npm run clean",
```

Comment:
Suggestion (follow-up, optional): npm run scripts don't allow for inline comments, but it would be nice to document the intended use of these scripts for contributors and maintainers. The README.md is a good start for running them.